packages/contracts/contracts/voting/token/TokenVotingSetup.sol

```diff
@@ -132,7 +130,7 @@ contract TokenVotingSetup is PluginSetup {
                 // it's simply ERC20 which gets checked by _isERC20
                 // Currently, not a satisfiable check..
```

Comment:
Don't we want to remove the `// Currently, not a satisfiable check..`?

Reply:
Let's leave it. It's true and helps other devs get the point.